src/packages/client/src/__tests__/integration/errors/connection-limit/test.ts

```diff
@@ -0,0 +1,34 @@
+import { getTestClient } from '../../../../utils/getTestClient'
+
+describe('connection-limit', () => {
+  expect.assertions(1) // cannot have any clients currently connected to db.
+  let clients: Array<any> = []
+
+  afterAll(async () => {
+    for (const client of clients) {
+      await client.$disconnect()
+    }
+  })
+
+  test('connection', async () => {
+    for (let i = 0; i <= 100; i++) {
+      const prismaClient = await getTestClient()
```

Comment:
This can be done outside the for loop.
I would also call it uppercase `PrismaClient` instead of `prismaClient`